pkg/diff/diff_test.go

```diff
@@ -178,10 +150,10 @@ func TestDiff(t *testing.T) {
 	leftDep := newDeployment()
 	leftUn := mustToUnstructured(leftDep)
 
-	diffRes := diff(t, leftUn, leftUn, GetDefaultDiffOptions())
+	diffRes := diff(t, leftUn, leftUn, diffOptionsForTest()...)
 	assert.False(t, diffRes.Modified)
 	ascii, err := printDiff(diffRes)
-	assert.Nil(t, err)
+	require.NoError(t, err)
```

Comment:
I replaced all `assert.Nil(t, err)` in this file because it was really hard to see what is wrong. Test should stop on first error to avoid executing more assertions and printing not so relevant messages (red herring!).